### ISSUE TITLE
Advanced Search Page: Keyboard Navigation / Tab Order 

### ISSUE THREAD

**Issue by KTottenDC** [DouglasCountyCO/Accessibility-PowerBI-Reports-Public#6]
Tab Order (current defaults)

1. Back to Search (Button)
2. Details | Mailing List | Redemption | FAQ (Button Group - no access to buttons)
3. Property Details
4. Redemption (?) -- this was not visible until using the Keyboard nav - might be a stray visual that is actually hidden.
5. Links (Group Box)
6. Map (Group Box)
7. Post Sale (Group Box ?)

Also of note is that the blue banner doesn't show up so there is no recognition of a page title like "Foreclosure Search Details"
Maybe not a problem since the person should know that they hit the link from the previous page.
